hashers.go

```diff
@@ -2,6 +2,7 @@ package trillian
 
 import (
 	"crypto"
+	_ "crypto/sha256"
```

Comment:
I thought the hash algorithms only get linked if imported somewhere. Not sure if SHA256 is guaranteed to be present.